mesos_slave/datadog_checks/mesos_slave/mesos_slave.py

```diff
@@ -136,23 +136,31 @@ def _get_json(self, url, timeout, verify, tags=None):
         return r.json()
 
     def _get_state(self, url, timeout, verify, tags):
-        return self._get_json(url + '/state.json', timeout, verify, tags)
+        try:
+            # Mesos version >= 0.25
+            endpoint = url + '/state'
+            master_state = self._get_json(endpoint, timeout, verify, failure_expected=True, tags=tags)
+        except CheckException as e:
+            self.log.warning('Encountered error getting state at {0}, message: {1}'.format(endpoint, str(e)))
```

Comment:
I think like this should not be a warning, maybe something like
```suggestion
            old_endpoint = endpoint + '.json'
            self.log.info('Unable to fetch state from {}. Retrying with the deprecated endpoint: {}'.format(endpoint, old_endpoint))
```